subsys/net/lib/dns/resolve.c

```diff
@@ -464,14 +440,12 @@ static inline int get_slot_by_id(struct dns_resolve_context *ctx,
 #if !defined(CONFIG_NET_TEST)
 static
 #endif
-int dns_validate_msg(struct dns_resolve_context *ctx,
-		     struct dns_msg_t *dns_msg,
-		     uint16_t *dns_id,
-		     int *query_idx,
-		     struct net_buf *dns_cname,
-		     uint16_t *query_hash)
+	int
+	dns_validate_msg(struct dns_resolve_context *ctx, struct dns_msg_t *dns_msg,
```

Comment:
This indentation looks weird, why is the type indented? Anyway, a general note, it's a bit hard to review due to all those formatting changes, hard to spot the actual meaningful ones. Can we drop it from the PR?

Reply:
Fixed. Removed formatting changes.